### DISCUSSION TITLE
Running on LAN with no internet connection

### DISCUSSION THREAD

**Discussion by dom-robinson** [mynaparrot/plugNmeet-server#516] [Q&A]
Hi folks - at the end of this week I am looking to run a conference call on a train (!) I need a conference server that can run on a small computer / raspberry pi / laptop, with a WiFi router, and other passengers can join, find the conference call and interact with the host. All participants will be on the train. Looking at plugnmeet setup it requires some external dns. Can it be setup on a lan with no external internet??

**Comment by jibon57** [comment]
Hi,

Setting up livekit in local network is possible but little bit tricky. You'll have to setup it manually. Also for video conference require bit more powerful server/device.

Thanks
